tools/spinel-cli/SNIFFER.md

```diff
@@ -0,0 +1,94 @@
+# Spinel Sniffer Reference
+
+Any Spinel NCP node can be made into a promiscuous packet sniffer, and this 
+tool both intializes a device into this mode and outputs a pcap stream that
+can be saved or piped directly into Wireshark.
+
+## System Requirements
+
+| OS     | Minimum Version  |
```

Comment:
Just say support platforms.  I'm sure this runs in other environments and these are not necessarily the minimum requirements.